src/frisby/expects.js

```diff
@@ -100,8 +100,13 @@ const expects = {
 
     incrementAssertionCount();
 
+    let options = { allowUnknown: true };
+    if (path) {
+      options.language = { label: path };
```

Comment:
@vlucas 

There is no effect multi-line json and jsonType expectations.

i.e.:
```js
  it('issue 425', done => {
    frisby.fromJSON({
      data: {
        name: 123,
        email: 'abc@abc.com',
      }
    })
      .expect('status', 200)
      .expect('jsonTypes', 'data', {
        name: Joi.string().required(),
        email: Joi.string().email().required(),
      })
      .done(done);
  });
```

**Error message**
```
ValidationError: child "name" fails because ["name" must be a string]
```